plugins/richcombo/plugin.js

```diff
@@ -96,6 +96,14 @@ CKEDITOR.plugins.add( 'richcombo', {
 				panelDefinition: panelDefinition,
 				items: {}
 			};
+
+			/**
+			 * Array containing richCombos registered listeners.
+			 *
+			 * @type {Array}
+			 * @private
+			 */
+			this._.listeners = [];
```

Comment:
If we add it to `_` namespace, API docs are not needed. It would be even weird to document only one property of it. Let's just add `listeners` array to the object literal above.

Reply:
I guess every piece of extra information might be helpful in future especially if it's already written, but I guess this property is so obvious we can drop docs, and move it to the object literal.